### ISSUE TITLE
Destinations to Adobe Systems

### ISSUE THREAD

**Issue by nhecht** [AdobeDocs/experience-platform.en#32]
Issue in help/rtcdp/destinations/adobe-destinations.md

We should add AAM, Target, Ad Cloud as possible destinations in the destination documentation. 
I believe this could just mostly follow the content provided in the segment sharing documentation. But customers are expecting to see it as a destination.
https://docs.adobe.com/help/en/audience-manager/user-guide/implementation-integration-guides/integration-experience-platform/aam-aep-audience-sharing.html

**Comment by djwendo** [comment]
Thanks @nhecht , this is good feedback. @vgiurgiu tagging you so that you can create a ticket to work on this.

**Comment by vgiurgiu** [comment]
Thanks both. I created https://jira.corp.adobe.com/browse/PLAT-71550 to track the work. 
We'll need to discuss with the destinations product team how best to surface this information in the destinations catalog. 

I will reply on this thread after I make the updates.

**Comment by vgiurgiu** [comment]
@nhecht - we made the following updates following to address this issue:

* [Adobe destinations page](https://docs.adobe.com/help/en/experience-platform/rtcdp/destinations/destinations-cat/adobe-destinations/adobe-destinations.html) - we divided the destinations by type and added a new section for the segment sharing destinations you mentioned (Adobe Target, Ad Cloud, Audience Manager).

* We added a link in the left navigation, under Adobe destinations, to the Experience Platform segment sharing page which describes the integration with the aforementioned destinations.

* [Experience Platform segment sharing to Audience Manager and other solutions](https://docs.adobe.com/help/en/audience-manager/user-guide/implementation-integration-guides/integration-experience-platform/aam-aep-audience-sharing.html) - we added an _important_ note to clarify that an Audience Manager license is **not** necessary in order to share Experience Platform segments to the Experience Cloud solutions you mentioned.
